install/helm/open-match/subcharts/open-match-test/templates/e2e-job.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: e2e-job
+  namespace: {{ .Release.Namespace }}
+  annotations: {{- include "openmatch.chartmeta" . | nindent 4 }}
+  labels:
+    app: {{ template "openmatch.name" . }}
+    component: e2e-job
+    release: {{ .Release.Name }}
+spec:
+  selector:
+    app: {{ template "openmatch.name" . }}
+    component: e2e-job
+  # Specifies the number of retries before marking this job failed. Defaults to 6
```

Comment:
6? or 3?

Reply:
Oh I mean default is 6 if we don't specify the number of retries at here